src/main/java/bio/terra/app/controller/ApiValidationExceptionHandler.java

```diff
@@ -34,29 +35,29 @@ protected ResponseEntity<Object> handleExceptionInternal(
       Exception ex,
       @Nullable Object body,
       HttpHeaders headers,
-      HttpStatus status,
+      HttpStatusCode status,
       WebRequest request) {
 
-    if (HttpStatus.INTERNAL_SERVER_ERROR == status) {
+    if (status.is5xxServerError()) {
       request.setAttribute(WebUtils.ERROR_EXCEPTION_ATTRIBUTE, ex, WebRequest.SCOPE_REQUEST);
     }
-
-    Object responseBody = body;
-    if (responseBody == null) {
+    var responseBody = body;
+    // Without specifically grabbing the exception message for validation errors,
+    // TDR will swallow the error and return a 400 with no error message.
+    if (responseBody == null || status.isSameCodeAs(HttpStatus.BAD_REQUEST)) {
       responseBody =
           new ErrorModel()
               .message(status + " - see error details")
               .addErrorDetailItem(ex.getMessage());
     }
-
     return new ResponseEntity<>(responseBody, headers, status);
```

Comment:
With these changes, a ProblemDetail is returned for the response body, making the responseBody no longer null for bad request exceptions. So, I've added a special case for BAD_REQUESTS to continue the work originally added in this PR: https://github.com/DataBiosphere/jade-data-repo/pull/869